src/cluecode/finder.py

```diff
@@ -45,6 +45,21 @@
 """
 
 
+def prefilter_urls_and_emails(key, line):
+    """
+    A function that acts as a prefilter for the URL and Email detection which
+    prevents the lines from getting parsed by the complex regex to detect the
+    clues.
+    """
+    prefilter_dict = {'emails': ['@'],
```

Comment:
what about ftp and other possible urls?
Also as said above tow functions are likely clearer

Reply:
Fixed, thanks ! :)